src/object/schema_manager.c

```diff
@@ -422,7 +418,7 @@ static bool sm_is_possible_to_recreate_constraint (MOP class_mop, const SM_CLASS
 static bool sm_filter_index_pred_have_invalid_attrs (SM_CLASS_CONSTRAINT * constraint, char *class_name,
 						     SM_ATTRIBUTE * old_atts, SM_ATTRIBUTE * new_atts);
 
-static int sm_collect_truncatable_classes (MOP class_mop, unordered_oid_set & trun_classes, bool is_cascade);
+static int sm_collect_truncatable_classes (MOP class_mop, std::unordered_set < OID > &trun_classes, bool is_cascade);
```

Comment:
Personal curious:

Why do you change the typename T from OID* to OID? Is there any benefit?

Reply:
I think value is much safer and simpler than pointer, and OID* and OID have the same size in 64bit.

Reply:
I got it.